skeleton/src/main/views/webpack/js.njk

```diff
@@ -1 +1 @@
-<script src="/main-dev.js"></script>
+ <script src="main-dev.js"></script>
```

Comment:
why change this?

Reply:
think I added the wrong file to the pull request, let me fix apologies